test-app/package.json

```diff
@@ -0,0 +1,137 @@
+{
+  "name": "test-app",
+  "version": "6.1.0",
+  "description": "Bootstrap components for Ember.js",
+  "keywords": [
+    "bootstrap"
+  ],
+  "homepage": "https://www.ember-bootstrap.com/",
+  "bugs": {
+    "url": "https://github.com/ember-bootstrap/ember-bootstrap/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/ember-bootstrap/ember-bootstrap"
+  },
+  "license": "MIT",
+  "author": "Simon Ihmig <ihmig@kaliber5.de>",
+  "directories": {
+    "doc": "docs",
+    "test": "tests"
+  },
+  "scripts": {
+    "build": "ember build --environment=production",
+    "changelog": "lerna-changelog",
+    "lint": "concurrently \"npm:lint:*(!fix)\" --names \"lint:\"",
+    "lint:fix": "concurrently \"npm:lint:*:fix\" --names \"fix:\"",
+    "lint:hbs": "ember-template-lint .",
+    "lint:hbs:fix": "ember-template-lint . --fix",
+    "lint:js": "eslint . --cache",
+    "lint:js:fix": "eslint . --fix",
+    "nodetest": "mocha node-tests --recursive",
+    "start": "ember serve",
+    "testa": "concurrently \"npm:lint\" \"npm:test:*\" --names \"lint,test:\"",
+    "test:ember": "ember test",
+    "test:ember-compatibility": "ember try:each"
+  },
+  "dependencies": {
```

Comment:
Apps should have `devDependencies` only, shouldn't they?

Reply:
I think they can have both, but to be sure I've added all of them to devDependencies